<html>
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" dtd-version="3.0" xml:lang="en" article-type="research article">
<front>
<journal-meta>
<journal-id journal-id-type="publisher-id">Frontiers</journal-id>
<journal-title-group>
<journal-title>Frontiers in Immunology</journal-title>
</journal-title-group>
<issn pub-type="epub">2164-5388</issn>
<publisher>
<publisher-name>Scientific Research Publishing</publisher-name>
</publisher>
</journal-meta>
<article-meta>
<article-id pub-id-type="doi">10.4236/ojbiphy.2014.44016</article-id>
<article-id pub-id-type="publisher-id">OJBIPHY-51126</article-id>
<article-categories>
<subj-group subj-group-type="heading">
<subject>Articles</subject>
</subj-group>
<subj-group subj-group-type="Discipline-v2">
<subject>Physics&Mathematics</subject>
</subj-group>
</article-categories>
<title-group>
<article-title>
Human Prion Protein Conformational Changes Susceptibility: A Molecular Dynamics Simulation Study
</article-title>
</title-group>
<contrib-group>
<contrib contrib-type="author" xlink:type="simple">
<name name-style="western">
<surname>aura</surname>
<given-names>Alejandra Mandujano-Rosas</given-names>
</name>
<xref ref-type="aff" rid="aff1">
<sup>1</sup>
</xref>
</contrib>
<contrib contrib-type="author" xlink:type="simple">
<name name-style="western">
<surname>Daniel</surname>
<given-names>Osorio-González</given-names>
</name>
<xref ref-type="aff" rid="aff1">
<sup>1</sup>
</xref>
<xref ref-type="corresp" rid="cor1">
<sup>*</sup>
</xref>
</contrib>
<contrib contrib-type="author" xlink:type="simple">
<name name-style="western">
<surname>Pedro</surname>
<given-names>Guillermo Reyes-Romero</given-names>
</name>
<xref ref-type="aff" rid="aff2">
<sup>2</sup>
</xref>
</contrib>
<contrib contrib-type="author" xlink:type="simple">
<name name-style="western">
<surname>Jorge</surname>
<given-names>Mulia-Rodríguez</given-names>
</name>
<xref ref-type="aff" rid="aff1">
<sup>1</sup>
</xref>
</contrib>
</contrib-group>
<aff id="aff1">
<addr-line>
Laboratory of Molecular Biophysics, Faculty of Sciences, Autonomous University of State of Mexico, Toluca, Mexico
</addr-line>
</aff>
<aff id="aff2">
<addr-line>
Laboratory of Advaced Physics, Faculty of Sciences, Autonomous University of State of Mexico, Toluca, Mexico
</addr-line>
</aff>
<author-notes>
<corresp id="cor1">
* E-mail:
<email>dog@uaemex.mx(DO)</email>
;
</corresp>
</author-notes>
<pub-date pub-type="epub">
<day>23</day>
<month>09</month>
<year>2014</year>
</pub-date>
<volume>04</volume>
<issue>04</issue>
<fpage>169</fpage>
<lpage>175</lpage>
<history>
<date date-type="received">
<day>29</day>
<month>August</month>
<year>2014</year>
</date>
<date date-type="rev-recd">
<day>30</day>
<month>September</month>
<year>2014</year>
</date>
<date date-type="accepted">
<day>29</day>
<month>October</month>
<year>2014</year>
</date>
</history>
<permissions>
<copyright-statement>
© Copyright 2014 by authors and Scientific Research Publishing Inc.
</copyright-statement>
<copyright-year>2014</copyright-year>
<license>
<license-p>
This work is licensed under the Creative Commons Attribution International License (CC BY). http://creativecommons.org/licenses/by/4.0/
</license-p>
</license>
</permissions>
<abstract>
<p>
Prion proteins are related to the development of incurable and invariably fatal neurodegenerative diseases in humans and animals. The pathogenicity involves the conversion of the host-encoded-alpha rich isoform of prion protein, PrP
<sup>C</sup>
, into a misfolded beta-strand rich conformer, PrP
<sup>Sc</sup>
. Although it has already been described that many punctual mutations alter the stability of PrP
<sup>C</sup>
, making it more prone to adopt an abnormal misfolded structure, the majority of cases reported among general population are sporadic in wild-type organisms. Thus, in this work we studied the dynamics and stability profiles of wild-type human prion protein by Molecular Dynamics (MD) simulation at different solvent temperatures. This analysis brought out certain residues and segments of the prion protein as critical to conformational changes; these results are consistent with experimental reports showing that protein mutants in those positions are related to the development of disease.
</p>
</abstract>
<kwd-group>
<kwd>Prion Protein</kwd>
<kwd>Misfolding Susceptibility</kwd>
<kwd>Molecular Dynamics Simulation</kwd>
</kwd-group>
</article-meta>
</front>
<body>
<sec id="s1">
<title>1. Introduction</title>
<p>
The human prion protein (hPrP) is a membrane N-linked glycosylphosphatidylinositol-anchored glycoprotein that has 253 amino acids when it is newly synthesized. It is expressed in various tissues, but is particularly present in nervous system. Its primary structure includes an N-terminal 22-residue signal peptide, which mediates translocation into the endoplasmic reticulum [
<xref ref-type="bibr" rid="scirp.51126-ref1">1</xref>
] , and is cleaved during importation. Soon after importation event, core glycans are attached to N181 and N197, and also a glycosylphosphatidylinositol anchor is trans- ferred to S230 [
<xref ref-type="bibr" rid="scirp.51126-ref2">2</xref>
] [
<xref ref-type="bibr" rid="scirp.51126-ref3">3</xref>
] . Core glycans are subjected to further processing in the Golgi compartment [
<xref ref-type="bibr" rid="scirp.51126-ref4">4</xref>
] . hPrP also has oxidation-susceptible methionines at positions 129, 134, 154, 166, 205, 206, 231 [
<xref ref-type="bibr" rid="scirp.51126-ref5">5</xref>
] . Two cysteine residues at positions 179 and 214 are linked by a single disulfide bond [
<xref ref-type="bibr" rid="scirp.51126-ref6">6</xref>
] . The last C-terminal 22 amino acid residues comprise another signal sequence, which is also absent in the 209-residue mature form of the protein [
<xref ref-type="bibr" rid="scirp.51126-ref1">1</xref>
] .
</p>
<p>
Sequence PQGGGGWGQ is among important structural regions of hPrP, beginning at residue number 51, and followed by four repetitions of the octapeptide PHGG(G/S)WGQ which encompasses residues 60 to 91; such octapeptides are cofactor binding sites. It also has a palindromic sequence AGAAAAGAA that spans residues 113 to 120, as part of a downstream tightly packed hydrophobic core. Residues at positions 127 and 129 are polymorphic, as it could be either glycine or valine in the first case, and methionine or valine in the second one [
<xref ref-type="bibr" rid="scirp.51126-ref7">7</xref>
] .
</p>
<p>
In accordance with the protein-only hypothesis, a misfolding of human prion protein is the cause of prion diseases. Experimental evidence supports that this is the sole etiologic agent of a group of neurodegenerative diseases related to the conversion of the native isoform, an alpha-helix rich cellular protein (PrP
<sup>C</sup>
), into a beta- strand rich misfolded conformer, named “Scrapie” prion protein (PrP
<sup>Sc</sup>
) [
<xref ref-type="bibr" rid="scirp.51126-ref8">8</xref>
] . The PrP
<sup>Sc</sup>
is thought to be infectious through nucleation of complete or partial PrP
<sup>Sc</sup>
forming rod-like aggregates. Template assistance and nucleated polymerization models have been proposed to explain the mechanisms of conformational conversion and aggregation, but they are still poorly understood [
<xref ref-type="bibr" rid="scirp.51126-ref5">5</xref>
] . Thus, this work is aimed to provide insight into them by using a molecular dynamics approximation to the problems.
</p>
<p>
PrP
<sup>C</sup>
has a flexibly disordered N-terminal tail and a structured C-terminal globular domain from amino acid 125 to 231. The C-terminal globular domain includes three alpha-helixes known as H1, H2 and H3 at 144 - 154, 173 - 194 and 200 - 228 positions, and a short antiparallel beta-strand region at 127 - 130 and 161 - 164 residues (S1 and S2). It also has three zones of structural irregularity at 167 - 171, 187 - 194 and 219 - 228 [
<xref ref-type="bibr" rid="scirp.51126-ref9">9</xref>
] . H1 and H2 are linked by a loop formed by 165 - 171 residues, better known as X-loop. The protein hydrophobic core comprises interactions between H2 and H3 and between H3 and the loop preceding H1 (S1-H1 loop), as well as contacts between the beta-strand with the rest of the globular domain [
<xref ref-type="bibr" rid="scirp.51126-ref5">5</xref>
] .
</p>
<p>
More than 30 different punctual mutations in the open reading frame of the human prion gene (PRNP) have been studied as they are linked to human prion inherited diseases, such as Gerstmann-Sträussler-Scheinker syndrome, fatal familial insomnia and familial Creutzfeldt-Jakob disease. All these diseases have variations in clinical, physiological and pathological characteristics, which are probably explained by the differences in the mechanism of destabilization of PrP
<sup>C</sup>
, conversion into PrP
<sup>Sc</sup>
, aggregation, and by the specific brain regions where all these processes occur. It is particularly remarkable that many missense point mutations are clustered within H2 and H3, and the loop between them. Mutations leading to disease have also been reported, although much less represented, at N-terminal domain, particularly at the octapeptide region and the disordered zone before S1 [
<xref ref-type="bibr" rid="scirp.51126-ref10">10</xref>
] - [
<xref ref-type="bibr" rid="scirp.51126-ref14">14</xref>
] .
</p>
<p>
Mutations may alter either the thermodynamic stability of PrP
<sup>C</sup>
, change surface properties that in turn modify interactions, and vary the response to cellular pathways control processes, or a combination of these mechanisms. However, solution structures of hPrP mutants are conformationally similar to the wild-type protein, and only molecular simulation techniques have been capable of elucidate differences between them [
<xref ref-type="bibr" rid="scirp.51126-ref15">15</xref>
] - [
<xref ref-type="bibr" rid="scirp.51126-ref18">18</xref>
] .
</p>
<p>
<xref ref-type="table" rid="table1">Table 1</xref>
summarizes different mutations associated with the development of human prion inherited diseases.
</p>
</sec>
<sec id="s2">
<title>2. Material and Methods</title>
<p>
Extensive all-atom molecular dynamics simulations at 37˚C and different pH values were made to study the structure of wild-type human prion protein, first in the vacuum, and then using water as solvent. We used spatial coordinates of human prion protein structured region (amino acid residues 125 - 228) as in 1QLZ code of Protein Data Bank.
</p>
<p>
In order to relax the protein in vacuum, MD was applied at minimum energy conditions and under the influence of GROMOS96 force field at GROMACS 4.0.7 suite of programs [
<xref ref-type="bibr" rid="scirp.51126-ref19">19</xref>
] . Consecutively, periodic frontier conditions were incorporated by using a rhombic dodecahedral water box which volume was 283 nm
<sup>3</sup>
. This process was repeated using 7778 water molecules as solvent with Simple Point Charge (SPC) model.
</p>
<table-wrap id="table1">
<label>
<xref ref-type="table" rid="table1">Table 1</xref>
</label>
<caption>
<title>
This table shows the most important structural and punctual mutations reported to have influence over the con- formational stability/pathological capacity of human prion protein
</title>
</caption>
<table>
<tbody>
<thead>
<tr>
<th align="center" valign="middle">Mutation</th>
<th align="center" valign="middle">Location</th>
<th align="center" valign="middle">Proposed alteration mechanism</th>
<th align="center" valign="middle">Remarks</th>
</tr>
</thead>
<tr>
<td align="center" valign="middle">
Insertion of additional octapeptide repeats at N-terminus
</td>
<td align="center" valign="middle">N-terminal domain</td>
<td align="center" valign="middle">
The insertion modifies copper and glycosaminoglycan binding affinities [
<xref ref-type="bibr" rid="scirp.51126-ref18">18</xref>
] .
</td>
<td align="center" valign="middle">-</td>
</tr>
<tr>
<td align="center" valign="middle">G114V, A117V, and G131V</td>
<td align="center" valign="middle">Hydrophobic core</td>
<td align="center" valign="middle">
Mutations showed an effect on translocation, resulting in slightly increased generation of a topological particular transmembrane form [
<xref ref-type="bibr" rid="scirp.51126-ref5">5</xref>
] .
</td>
<td align="center" valign="middle">-</td>
</tr>
<tr>
<td align="center" valign="middle">D178N</td>
<td align="center" valign="middle">H2</td>
<td align="center" valign="middle">
Abolishment of a salt bridge between D178-R164 and possibly of hydrogen bond interactions with Y128 and Y169. Reduction of thermodynamic stability of hPrP and promotion of aggregation [
<xref ref-type="bibr" rid="scirp.51126-ref14">14</xref>
] .
</td>
<td align="center" valign="middle">
It causes fatal familial insomnia in combination with M129 and familial Creutzfeldt-Jakob disease with V129.
</td>
</tr>
<tr>
<td align="center" valign="middle">C179A, C214A</td>
<td align="center" valign="middle">Disulfide bridge</td>
<td align="center" valign="middle">
Mutants are insoluble and form amorphous aggregates [
<xref ref-type="bibr" rid="scirp.51126-ref6">6</xref>
] .
</td>
<td align="center" valign="middle">-</td>
</tr>
<tr>
<td align="center" valign="middle">T183A</td>
<td align="center" valign="middle">H2</td>
<td align="center" valign="middle">
Disruption of hydrogen bond interaction with Y162. Increment of the flexibility of the globular domain. Reduction of thermodynamic stability of hPrP [
<xref ref-type="bibr" rid="scirp.51126-ref14">14</xref>
] .
</td>
<td align="center" valign="middle">It causes familial Creutzfeldt-Jakob disease.</td>
</tr>
<tr>
<td align="center" valign="middle">H187R</td>
<td align="center" valign="middle">H2</td>
<td align="center" valign="middle">
Introduction of the positively charged side chain increments electrostatic repulsion between Arg156 and Arg187, which drives both side chains away from their original positions. This can lead to the solvent exposure of hydrophobic core [
<xref ref-type="bibr" rid="scirp.51126-ref20">20</xref>
] .
</td>
<td align="center" valign="middle">Related to Gerstmann-Sträussler-Scheinker syndrome</td>
</tr>
<tr>
<td align="center" valign="middle">E196K</td>
<td align="center" valign="middle">H2-H3 loop</td>
<td align="center" valign="middle">
Loss of a salt bridge between E196-R156, which destabilizes the F198 hydrophobic pocket. HA detachment of the short helix (H1) from the core, exposure of side chain F198, and formation of a nonnative strand at the N-terminus [
<xref ref-type="bibr" rid="scirp.51126-ref19">19</xref>
] .
</td>
<td align="center" valign="middle">It causes familial Creutzfeldt-Jakob disease.</td>
</tr>
<tr>
<td align="center" valign="middle">F198S</td>
<td align="center" valign="middle">Hydrophobic core</td>
<td align="center" valign="middle">
Substitution of lateral chains leaves a gap in the hydrophobic core between H2 and H3. Increment of the flexibility of the globular domain, more strongly in loop between H2 and H3 [
<xref ref-type="bibr" rid="scirp.51126-ref20">20</xref>
] .
</td>
<td align="center" valign="middle">-</td>
</tr>
<tr>
<td align="center" valign="middle">V210I, Q212P</td>
<td align="center" valign="middle">H3</td>
<td align="center" valign="middle">
The mutations increase structural disorder of the S2-H2 loop, and rise distance between S2-H2 loop and H3 which, in turn, increment the exposure of hydrophobic residues to solvent [
<xref ref-type="bibr" rid="scirp.51126-ref19">19</xref>
] [
<xref ref-type="bibr" rid="scirp.51126-ref20">20</xref>
] .
</td>
<td align="center" valign="middle">
V210I causes familial Creutzfeldt-Jakob disease, and Q212P causes Gerstmann- Sträussler-Scheinker syndrome
</td>
</tr>
<tr>
<td align="center" valign="middle">V180I, V203I, and V210I</td>
<td align="center" valign="middle">Hydrophobic core</td>
<td align="center" valign="middle">
Addition of an extra methylene group in lateral chains cause steric crowding. Reduction of thermodynamic stability. V180I and V210I change hydrophobic packing of H2-H3 loop residues [
<xref ref-type="bibr" rid="scirp.51126-ref5">5</xref>
] .
</td>
<td align="center" valign="middle">-</td>
</tr>
<tr>
<td align="center" valign="middle">E200K</td>
<td align="center" valign="middle">H3</td>
<td align="center" valign="middle">
The mutation modifies surface charges which influence the electrostatic interactions with other cofactors [
<xref ref-type="bibr" rid="scirp.51126-ref5">5</xref>
] .
</td>
<td align="center" valign="middle">It causes familial Creutzfeldt-Jakob disease.</td>
</tr>
<tr>
<td align="center" valign="middle">Y218N</td>
<td align="center" valign="middle">H3</td>
<td align="center" valign="middle">
Reduction of hydrophobic packing around the X-loop. Increase of the H2-H3 inter helical angle, which in turn disrupts the packing around F198 [
<xref ref-type="bibr" rid="scirp.51126-ref19">19</xref>
] . Formation of a nonnative contact between E221 and S132 on the S1-HA loop [
<xref ref-type="bibr" rid="scirp.51126-ref19">19</xref>
] .
</td>
<td align="center" valign="middle">It causes Gerstmann-Sträussler- Scheinker syndrome</td>
</tr>
</tbody>
</table>
</table-wrap>
<p>
The system protein-water was neutralized by addition of sodium and chloride ions, and the potential energy was defined as the sum of two types of non-bonded interactions (electrostatic and van der Waals) and three types of bonded interactions (chemical, angles and twisting).
</p>
<p>
The equilibration process was started with a set of velocities taken from a Maxwell-Boltzmann distribution until it reached 305 K in about 350 ps using the Berendsen thermostat for both systems (protein and solvent). The production process was implemented with the same thermostat and the Parrinello-Rahman barostat; the total simulation length was 80 ns. The performance of computer system was 5.0 ns per day.
</p>
<p>
The measure of the deviation between the position of particle
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x6.png" xlink:type="simple"/>
</inline-formula>
and some reference position was calculated with the Root Mean Square Fluctuation (RMSF).
</p>
<disp-formula id="scirp.51126-formula1897">
<label>(1)</label>
<graphic position="anchor" xlink:href="http://html.scirp.org/file/5-1850108x7.png" xlink:type="simple"/>
</disp-formula>
<p>
where
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x8.png" xlink:type="simple"/>
</inline-formula>
is the time over one wants to calculate the average,
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x9.png" xlink:type="simple"/>
</inline-formula>
is the position of the particle
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x10.png" xlink:type="simple"/>
</inline-formula>
on the time
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x11.png" xlink:type="simple"/>
</inline-formula>
and
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x12.png" xlink:type="simple"/>
</inline-formula>
is the reference position of the particle
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x13.png" xlink:type="simple"/>
</inline-formula>
. In order to obtain the relative vibrational motion of different regions of the protein we use the B-factor
</p>
<disp-formula id="scirp.51126-formula1898">
<label>(2)</label>
<graphic position="anchor" xlink:href="http://html.scirp.org/file/5-1850108x14.png" xlink:type="simple"/>
</disp-formula>
<p>
where
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x15.png" xlink:type="simple"/>
</inline-formula>
is the mean square displacement and is related with the self-diffusion coefficient
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x16.png" xlink:type="simple"/>
</inline-formula>
via the Einstein relation
</p>
<disp-formula id="scirp.51126-formula1899">
<label>(3)</label>
<graphic position="anchor" xlink:href="http://html.scirp.org/file/5-1850108x17.png" xlink:type="simple"/>
</disp-formula>
<p>
where
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x18.png" xlink:type="simple"/>
</inline-formula>
is the initial position of the particle
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x19.png" xlink:type="simple"/>
</inline-formula>
.
</p>
<p>
Also we used the
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x20.png" xlink:type="simple"/>
</inline-formula>
-component pressure tensor
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x20.png" xlink:type="simple"/>
</inline-formula>
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x21.png" xlink:type="simple"/>
</inline-formula>
to estimate the shear viscosity
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x20.png" xlink:type="simple"/>
</inline-formula>
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x21.png" xlink:type="simple"/>
</inline-formula>
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x22.png" xlink:type="simple"/>
</inline-formula>
which evaluates the rates of possible changes in the protein conformation, both variables are related by
</p>
<disp-formula id="scirp.51126-formula1900">
<label>(4)</label>
<graphic position="anchor" xlink:href="http://html.scirp.org/file/5-1850108x23.png" xlink:type="simple"/>
</disp-formula>
<p>
Here,
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x24.png" xlink:type="simple"/>
</inline-formula>
is the volume,
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x24.png" xlink:type="simple"/>
</inline-formula>
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x25.png" xlink:type="simple"/>
</inline-formula>
the temperature,
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x24.png" xlink:type="simple"/>
</inline-formula>
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x25.png" xlink:type="simple"/>
</inline-formula>
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x26.png" xlink:type="simple"/>
</inline-formula>
the initial time and
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x24.png" xlink:type="simple"/>
</inline-formula>
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x25.png" xlink:type="simple"/>
</inline-formula>
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x26.png" xlink:type="simple"/>
</inline-formula>
<inline-formula>
<inline-graphic xlink:href="http://html.scirp.org/file/5-1850108x27.png" xlink:type="simple"/>
</inline-formula>
the Boltzmann constant [
<xref ref-type="bibr" rid="scirp.51126-ref21">21</xref>
] [
<xref ref-type="bibr" rid="scirp.51126-ref22">22</xref>
] .
</p>
</sec>
<sec id="s3">
<title>3. Results and Discussion</title>
<p>
The solvated protein was stabilized with an initial temperature of 310 K, and then the temperature of the solvent was modified in order to stimulate the protein and identify the regions with higher vibrational states.
<xref ref-type="fig" rid="fig1">Figure 1</xref>
shows that such regions correspond to residues 144 - 149, 178 - 181, 193 - 197 and 221 - 223. The RMSF was calculated with (1).
</p>
<fig id="fig1" position="float">
<label>
<xref ref-type="fig" rid="fig1">Figure 1</xref>
</label>
<caption>
<title>
Root mean square fluctuations per residue show the most vulnerable regions to vibrations. Labeled temperatures correspond to solvent. The residues 150, 184 and 210 do not show substantial changes with respect to the solvent temperature
</title>
</caption>
<graphic mimetype="image" position="float" xlink:type="simple" xlink:href="http://html.scirp.org/file/5-1850108x28.png"/>
</fig>
<p>
In order to identify the relative vibrational motion of different parts of the protein, we calculated de B-factor for the same solvent temperatures, as can be observed in
<xref ref-type="fig" rid="fig2">Figure 2</xref>
.
</p>
<p>
It is well known that calculation of viscosity with (4) converges very slowly [
<xref ref-type="bibr" rid="scirp.51126-ref21">21</xref>
] ; however we were interested in analyzing its evolution in early times because its variations stimulate conformational changes on the protein.
<xref ref-type="fig" rid="fig3">Figure 3</xref>
shows that viscosity has partially uniform oscillations around 1.5 cP when the solvent has the lowest temperature, in other words, viscosity varies more uniformly at the lowest temperature than at other temperatures, maintaining values predominantly below 2 cP.
</p>
<fig id="fig2" position="float">
<label>
<xref ref-type="fig" rid="fig2">Figure 2</xref>
</label>
<caption>
<title>
The B-factor reveals the regions with more thermal effects on the protein. These regions are the same that can be seen in
<xref ref-type="fig" rid="fig1">Figure 1</xref>
. Labeled temperatures correspond to solvent
</title>
</caption>
<graphic mimetype="image" position="float" xlink:type="simple" xlink:href="http://html.scirp.org/file/5-1850108x29.png"/>
</fig>
<fig id="fig3" position="float">
<label>
<xref ref-type="fig" rid="fig3">Figure 3</xref>
</label>
<caption>
<title>
It shows the shear viscosity evolution in early times of production process simulation at three different solvent temperatures; it follows that the increasing-decreasing monotonic behavior when the solvent has a temperature lower than 310 K, is not a factor that interfere with the regions of increased flexibility in the protein
</title>
</caption>
<graphic mimetype="image" position="float" xlink:type="simple" xlink:href="http://html.scirp.org/file/5-1850108x30.png"/>
</fig>
</sec>
<sec id="s4">
<title>4. Conclusion</title>
<p>
The regions of high susceptibility to conformational changes are close to mutation sites experimentally tested and reported. According to
<xref ref-type="fig" rid="fig1">Figure 1</xref>
, the regions with higher vibrational states correspond to residues 144 - 149, 178 - 181, 193 - 197 and 221 - 223. The simulation showed important oscillations in nanosecond time scale order. These oscillations are not due to viscosity fluctuations, but due to temperature changes in solvent. The oscillations observed in three different properties indicate that the van der Waals term of the force field employed is very rigid, so we need to prove a more flexible potential with more realistic repulsion and attraction contributions, although it increases the computational time required for simulations.
</p>
</sec>
<sec id="s5">
<title>Acknowledgements</title>
<p>
The authors wish to thank Universidad Autónoma del Estado de México for financial support through project ID 3697/2014/CID.
</p>
</sec>
<sec id="s6">
<title>NOTES</title>
</sec>
</body>
<back>
<ref-list>
<title>References</title>
<ref id="scirp.51126-ref1">
<label>1</label>
<mixed-citation publication-type="other" xlink:type="simple">
Rane, N.S., Chakrabarti, O., Feigenbaum, L. and Hegde, R. (2010) Signal Sequence Insufficiency Contributes to Neurodegeneration Caused by Transmembrane Prion Protein. The Journal of Cell Biology, 188, 515-526. http://dx.doi.org/10.1083/jcb.200911115
</mixed-citation>
</ref>
<ref id="scirp.51126-ref2">
<label>2</label>
<mixed-citation publication-type="other" xlink:type="simple">
Haraguchi, T., Fisher, S., Olofsson, S., Endo, T., Groth, D., Tarentino, A., Borchelt, D.R., Teplow, D., Hood, L., Burlingame, A., Lycke, E., Kobata, A. and Prusiner, S.B. (1989) Asparagine-Linked Glycosylation of the Scrapie and Cellular Prion Proteins. Archives of Biochemistry and Biophysics, 274, 1-13. http://dx.doi.org/10.1016/0003-9861(89)90409-8
</mixed-citation>
</ref>
<ref id="scirp.51126-ref3">
<label>3</label>
<mixed-citation publication-type="other" xlink:type="simple">
Stahl, N., Borchelt, D.R., Hsiao, K. and Prusiner, S.B. (1987) Scrapie Prion Protein Contains a Phosphatidylinositol Glycolipid. Cell, 51, 229-240. http://dx.doi.org/10.1016/0092-8674(87)90150-4
</mixed-citation>
</ref>
<ref id="scirp.51126-ref4">
<label>4</label>
<mixed-citation publication-type="other" xlink:type="simple">
Stimson, E., Hope, J., Chong, A. and Burlingame, A.L. (1999) Site-Specific Characterization of the N-Linked Glycans of Murine Prion Protein by High-Performance Liquid Chromatography/Electrospray Mass Spectrometry and Exoglycosidase Digestions. Biochemistry, 38, 4885-4895. http://dx.doi.org/10.1021/bi982330q
</mixed-citation>
</ref>
<ref id="scirp.51126-ref5">
<label>5</label>
<mixed-citation publication-type="other" xlink:type="simple">
Zhou, Z. and Xiao, G. (2013) Conformational Conversion of Prion Protein in Prion Diseases. Acta Biochimica et BiophysicaSinica, 45, 465-476. http://dx.doi.org/10.1093/abbs/gmt027
</mixed-citation>
</ref>
<ref id="scirp.51126-ref6">
<label>6</label>
<mixed-citation publication-type="other" xlink:type="simple">
Ranjan-Maiti, N. and Surewicz, W.K. (2001) The Role of Disulfide Bridge in the Folding and Stability of the Recombinant Human Prion Protein. The Journal of Biological Chemistry, 276, 2427-2431. http://dx.doi.org/10.1074/jbc.M007862200
</mixed-citation>
</ref>
<ref id="scirp.51126-ref7">
<label>7</label>
<mixed-citation publication-type="other" xlink:type="simple">
Coleman, B.M., Harrison, C.F., Guo, B., Masters, C.L., Barnham, K.J., Lawson, V.A. and Hill, H.F. (2014) Pathogenic Mutations within the Hydrophobic Domain of the Prion Protein Lead to the Formation of the Protease-Sensitive Prion Species with Increased Lethality. Journal of Virology, 88, 2690-2703. http://dx.doi.org/10.1128/JVI.02720-13
</mixed-citation>
</ref>
<ref id="scirp.51126-ref8">
<label>8</label>
<mixed-citation publication-type="other" xlink:type="simple">
Prusiner, S.B. (1982) Novel Proteinaceous Infectious Particles Cause Scrapie. Science, 216, 136-144. http://dx.doi.org/10.1126/science.6801762
</mixed-citation>
</ref>
<ref id="scirp.51126-ref9">
<label>9</label>
<mixed-citation publication-type="other" xlink:type="simple">
Zahn, R., Liu, A., Luhrs, T., Riek, R., von Schroetter, C., Lopez Garcia, F., Billeter, M., Calzolai, L., Wider, G. and Wuthrich, K. (2000) NMR Solution Structure of the Human Prion Protein. Proceedings of the National Academy of Sciences U.S.A., 97, 145-150. http://dx.doi.org/10.1073/pnas.97.1.145
</mixed-citation>
</ref>
<ref id="scirp.51126-ref10">
<label>10</label>
<mixed-citation publication-type="other" xlink:type="simple">
Rossetti, G., Cong, X., Caliandro, R., Legname, G. and Carloni, P. (2011) Common Structural Traits Across Pathogenic Mutants of the Human Prion Protein and Their Implications for Familial Prion Disorders. Journal of Molecular Biology, 411, 700-712. http://dx.doi.org/10.1016/j.jmb.2011.06.008
</mixed-citation>
</ref>
<ref id="scirp.51126-ref11">
<label>11</label>
<mixed-citation publication-type="other" xlink:type="simple">
Kiachopoulos, S., Bracher, A., Winklhofer, K.F. and Talzet, J. (2005) Pathogenic Mutations Located at the Hydrophobic Core of the Prion Protein Interfere with Folding and Attachment of the Glycosylphosphatidylinositol Anchor. The Journal of Biological Chemistry, 280, 9320-9329. http://dx.doi.org/10.1074/jbc.M412525200
</mixed-citation>
</ref>
<ref id="scirp.51126-ref12">
<label>12</label>
<mixed-citation publication-type="other" xlink:type="simple">
Cheng, W., van der Kamp, M.W. and Daggett, V. (2014) Structural and Dynamic Properties of the Human Prion Protein. Biophysical Journal, 106, 1152-1163. http://search.proquest.com/docview/1508983494?accountid=37347 http://dx.doi.org/10.1016/j.bpj.2013.12.053
</mixed-citation>
</ref>
<ref id="scirp.51126-ref13">
<label>13</label>
<mixed-citation publication-type="other" xlink:type="simple">
Ilc, G., Giachin, G., Jaremko, M., Jaremko, L., Benetti, F., Plavec, J., Zhukov, I. and Legname, G. (2010) NMR Structure of the Human Prion Protein with the Pathological Q212P Mutation Reveals Unique Structural Features. PLoS ONE, 5, e11715. http://dx.doi.org/10.1371/journal.pone.0011715
</mixed-citation>
</ref>
<ref id="scirp.51126-ref14">
<label>14</label>
<mixed-citation publication-type="book" xlink:type="simple">
Tatzelt, J., Ed. (2011) Topics in Current Chemistry 305: Prion Proteins. Springer Berlin-Heidelberg, Berlin.
</mixed-citation>
</ref>
<ref id="scirp.51126-ref15">
<label>15</label>
<mixed-citation publication-type="other" xlink:type="simple">
Taylor, D.R., Whitehouse, I.J. and Hooper, N.M. (2009) Glypican-1 Mediates Both Prion Protein Lipid Raft Association and Disease Isoform Formation. PLoS Pathogens, 5, e1000666. http://dx.doi.org/10.1371/journal.ppat.1000666
</mixed-citation>
</ref>
<ref id="scirp.51126-ref16">
<label>16</label>
<mixed-citation publication-type="other" xlink:type="simple">
Elmallah, M.I.Y., Borgmeyer, U., Betzel, C. and Redecke, L. (2013) Impact of Methionine Oxidation as an Initial Event on the Pathway of Human Prion Protein Conversion. Prion, 7, 404-411. http://dx.doi.org/10.4161/pri.26745
</mixed-citation>
</ref>
<ref id="scirp.51126-ref17">
<label>17</label>
<mixed-citation publication-type="other" xlink:type="simple">
Mani, K., Cheng, F., Havsmark, B., Jonsson, M., Belting, M. and Fransson, L.A. (2003) Prion, Amyloid Beta-Derived Cu(II) Ions, or Free Zn(II) Ions Support S-Nitroso-Dependent Autocleavage of Glypican-1 Heparan Sulfate. The Journal of Biological Chemistry, 278, 38956-38965. http://dx.doi.org/10.1074/jbc.M300394200
</mixed-citation>
</ref>
<ref id="scirp.51126-ref18">
<label>18</label>
<mixed-citation publication-type="other" xlink:type="simple">
Wu, D., Zhang, W., Luo, Q., Luo, K., Huang, L., Wang, W., Huang, T., Chen, R., Lin, Y., Pang, D. and Xiao, G. (2010) Copper (II) Promotes the Formation of Soluble Neurotoxic PrP Oligomers in Acidic Environment. The Journal of Cell Biochemistry, 111, 627-633. http://dx.doi.org/10.1002/jcb.22743
</mixed-citation>
</ref>
<ref id="scirp.51126-ref19">
<label>19</label>
<mixed-citation publication-type="other" xlink:type="simple">
Zhong, L. (2010) Exposure of Hydrophobic Core in Human Prion Protein Pathogenic Mutant H187R. Journal of Biomolecular Structure Dynamics, 28, 355-361. http://dx.doi.org/10.1080/07391102.2010.10507365
</mixed-citation>
</ref>
<ref id="scirp.51126-ref20">
<label>20</label>
<mixed-citation publication-type="other" xlink:type="simple">
Cheng, C.J. and Dagget, V. (2014) Different Misfolding Mechanisms Converge on Common Conformational Changes: Human Prion Protein Pathogenic Mutants Y218N and E196K. Prion, 8, 125-135. http://dx.doi.org/10.4161/pri.27807
</mixed-citation>
</ref>
<ref id="scirp.51126-ref21">
<label>21</label>
<mixed-citation publication-type="other" xlink:type="simple">
Hess, B., Kutzner, C., van der Spoel, D. and Lindahl, E. (2008) GROMACS 4: Algorithms for Highly Efficient, Load Balanced and Scalable Molecular Simulation. Journal of Chemical Theory and Computation, 4, 435-447. http://dx.doi.org/10.1021/ct700301q
</mixed-citation>
</ref>
<ref id="scirp.51126-ref22">
<label>22</label>
<mixed-citation publication-type="other" xlink:type="simple">
Hess, B. (2002) Determining the Shear Viscosity of Model Liquids from Molecular Dynamics. The Journal of Chemical Physics, 116, 209-217. http://dx.doi.org/10.1063/1.1421362
</mixed-citation>
</ref>
</ref-list>
</back>
</article>
