<!DOCTYPE HTML>
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    

  <title>Publications</title>

  <meta name="author" content="Oscar Medina-Contreras">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  <link rel="stylesheet" type="text/css" href="stylesheet.css">
    <link rel="icon" href="Logo Aniversario.jpeg">
</head>
    
<body>
  <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    <tr style="padding:0px">
      <td style="padding:0px">
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
          <tr style="padding:0px">
            <td style="padding:2.5%;width:63%;vertical-align:middle">
              <p style="text-align:center">
                <h1>Papers</h1>
              </p>
		  
       <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
               
     
<tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#ffffff">
 <td style="padding:20px;width:25%;vertical-align:top">
   <div class="one">
    
     <img src='images/JP_2023.jpg' width="160">
   </div>
   
	 
    
      
         
     </td>
 <td style="padding:20px;width:75%;vertical-align:middle">
     <a style="text-decoration: none" href="https://www.sciencedirect.com/science/article/abs/pii/S1874391922003335?via%3Dihub">
         <papertitle>Immunoproteomics of cow's milk allergy in Mexican pediatric patients</papertitle>
   </a>
   <br>
   <p>Torres-Arroyo A, Martínez-Aguilar J, Castillo-Villanueva A, Zárate-Mondragón F, Cervantes-Bustamante R, Patiño-López G, Medina-Contreras O, Espinosa-Padilla SE, Valencia-Rojas S, Romero-Guzmán L, Oria-Hernández J, Reyes-Vivas H<p>
   
   <br>
   <em>J Proteomics </em>273, 104809 &nbsp <font color="red"><strong>(10.1016/j.jprot.2022.104809)</strong></font>
   <br>
 
 <a style="text-decoration: none" href="https://pubmed.ncbi.nlm.nih.gov/36587729/">PubMed</a>
   
   <p></p>
   <p>
       <p style="text-align:justify">
       Immunological mechanisms of non-IgE-mediated cow's milk protein allergy (CMPA) are not well understood. Such a circumstance requires attention with the aim of discovering new biomarkers that could lead to better diagnostic assays for early treatment. Here, we sought both to investigate the mechanism that underlies non-IgE-mediated CMPA and to identify cow's milk immunoreactive proteins in a Mexican pediatric patient group (n = 34). Hence, we determined the IgE and IgG1-4 subclass antibody levels against cow's milk proteins (CMP) by ELISA. Then, we performed 2D-Immunoblots using as first antibody immunoglobulins in the patients'serum that bound specifically against CMP together with CMP enrichment by ion-exchange chromatography. Immunoreactive proteins were identified by mass spectrometry-based proteomics. The serological test confirmed absence of specific IgE in the CMPA patients but showed significant increase in antigen-specific IgG1. Additionally, we identified 11 proteins that specifically bound to IgG1. We conclude that the detection of specific IgG1 together with an immunoproteomics approach is highly relevant to the understanding of CMPA's physiopathology and as a possible aid in making a prognosis since current evidence indicates IgG1 occurrence as an early signal of potential risk toward development of IgE-mediated food allergy. SIGNIFICANCE: Allergies are one of the most studied topics in the field of public health and novel protein allergens are found each year. Discovery of new principal and regional allergens has remarkable repercussions in precise molecular diagnostics, prognostics, and more specific immunotherapies. In this context, specific IgE is widely known to mediate physiopathology; however, allergies whose mechanism does not involve this immunoglobulin are poorly understood although their incidence has increased. Therefore, accurate diagnosis and adequate treatment are delayed with significant consequences on the health of pediatric patients. The study of type and subtypes of immunoglobulins associated with the immunoreactivity of cow's milk proteins together with an immunoproteomics approach allows better comprehension of physiopathology, brings the opportunity to discover new potential cow's milk protein allergens and may help in prognosis prediction (IgG1 occurrence as an early signal of possible risk toward development of IgE-mediated food allergy).
   
             </p>
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
    
      
<tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
  <td style="padding:20px;width:25%;vertical-align:top">
    <div class="one">
     
      <img src='images/JI_2016.png' width="160">
    </div>
    
    </script>
  </td>
  <td style="padding:20px;width:75%;vertical-align:middle">
    <a style="text-decoration: none"  href="https://journals.aai.org/jimmunol/article/196/1/34/105174/Cutting-Edge-IL-36-Receptor-Promotes-Resolution-of">
      <papertitle>IL-36 Receptor Promotes Resolution of Intestinal Damage</papertitle>
    </a>
    <br>
    <p>Medina-Contreras O, Harusato A, Nishio H, Flannigan KL, Ngo V, Leoni G, Neumann PA, Geem D, Lili LN, Ramadas RA, Chassaing B, Gewirtz AT, Kohlmeier JE, Parkos CA, Towne JE, Nusrat A, Denning TL<p>
    
    <br>
    <em>J Immunol </em>196(1), 34–8 &nbsp <font color="red"><strong>(10.4049/jimmunol.1501312)</strong></font>
    <br>
  
    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/26590314/">PubMed</a>
    
    <p></p>
    <p>
	    <p style="text-align:justify">
    IL-1 family members are central mediators of host defense. In this article, we show that the novel IL-1 family member IL-36γ was expressed during experimental colitis and human inflammatory bowel disease. Germ-free mice failed to induce IL-36γ in response to dextran sodium sulfate (DSS)-induced damage, suggesting that gut microbiota are involved in its induction. Surprisingly, IL-36R–deficient (Il1rl2−/−) mice exhibited defective recovery following DSS-induced damage and impaired closure of colonic mucosal biopsy wounds, which coincided with impaired neutrophil accumulation in the wound bed. Failure of Il1rl2−/− mice to recover from DSS-induced damage was associated with a profound reduction in IL-22 expression, particularly by colonic neutrophils. Defective recovery of Il1rl2−/− mice could be rescued by an aryl hydrocarbon receptor agonist, which was sufficient to restore IL-22 expression and promote full recovery from DSS-induced damage. These findings implicate the IL-36/IL-36R axis in the resolution of intestinal mucosal wounds.

    </p>
  </td>
</tr>
        </tbody></table>
            <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                <tr>
                    
              <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#FFFFFF">
                <td style="padding:20px;width:25%;vertical-align:top">
                  <div class="one">
                   
                    <img src='images/frontiers.png' width="160">
                  </div>
                  
                  </script>
                </td>
                <td style="padding:20px;width:75%;vertical-align:middle">
                  <a style="text-decoration: none"  href="https://www.frontiersin.org/articles/10.3389/fimmu.2022.979749/full">
                    <papertitle>IL-36γ is secreted through an unconventional pathway using the Gasdermin D and P2X7R membrane pores</papertitle>
                  </a>
                  <br>
                  <p>Manzanares-Meza LD, Gutiérrez-Román CI, Jiménez-Pineda A, Castro-Martínez F, Patiño-López G, Rodríguez-Arellano E, Valle-Rios R, Ortíz-Navarrete VF, Medina-Contreras O<p>
                  
                  <br>
                  <em>Front Immunol </em>13, 979749 &nbsp <font color="red"><strong>(10.3389/fimmu.2022.979749)</strong></font>
                  <br>
                
                  <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/36059446/">PubMed</a>
                  
                  <p></p>
                  <p>
                      <p style="text-align:justify">
                      Mucosal innate immunity functions as the first line of defense against invading pathogens. Members of the IL-1 family are key cytokines upregulated in the inflamed mucosa. Inflammatory cytokines are regulated by limiting their function and availability through their activation and secretion mechanisms. IL-1 cytokines secretion is affected by the lack of a signal peptide on their sequence, which prevents them from accessing the conventional protein secretion pathway; thus, they use unconventional protein secretion pathways. Here we show in mouse macrophages that LPS/ATP stimulation induces cytokine relocalization to the plasma membrane, and conventional secretion blockade using monensin or Brefeldin A triggers no IL-36γ accumulation within the cell. In silico modeling indicates IL-36γ can pass through both the P2X7R and Gasdermin D pores, and both IL-36γ, P2X7R and Gasdermin D mRNA are upregulated in inflammation; further, experimental blockade of these receptors' limits IL-36γ release. Our results demonstrate that IL-36γ is secreted mainly by an unconventional pathway through membrane pores formed by P2X7R and Gasdermin D.

                  </p>
                </td>
              </tr>
                  </tbody></table>
                      <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                          <tr>
<!--	</tbody></table>-->
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
            <tr>
                
          <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#ffffd0">
            <td style="padding:20px;width:25%;vertical-align:top">
              <div class="one">
               
                <img src='images/JCI_2011.jpeg' width="160">
              </div>
              
              </script>
            </td>
            <td style="padding:20px;width:75%;vertical-align:middle">
              <a style="text-decoration: none"  href="https://www.jci.org/articles/view/59150">
                <papertitle>CX3CR1 regulates intestinal macrophage homeostasis, bacterial translocation, and colitogenic Th17 responses in mice</papertitle>
              </a>
              <br>
              <p>Medina-Contreras O, Geem D, Laur O, Williams IR, Lira SA, Nusrat A, Parkos CA, Denning TL<p>
              
              <br>
              <em>J Clin Invest </em>121(12), 4787-95 &nbsp <font color="red"><strong>(10.1172/JCI59150)</strong></font>
              <br>
            
              <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/22045567/">PubMed</a>
              
              <p></p>
              <p>
                  <p style="text-align:justify">
                  The two most common forms of inflammatory bowel disease (IBD), Crohn's disease and ulcerative colitis, affect approximately 1 million people in the United States. Uncontrolled APC reactivity toward commensal bacteria is implicated in the pathogenesis of the disease. A number of functionally distinct APC populations exist in the mucosal lamina propria (LP) below the intestinal epithelium, but their relative contributions to inflammation remain unclear. Here, we demonstrate in mice important roles for the chemokine receptor CX3CR1 in maintaining LP macrophage populations, preventing translocation of commensal bacteria to mesenteric lymph nodes (mLNs), and limiting colitogenic Th17 responses. CX3CR1 was found to be expressed in resident LP macrophages (defined as CD11b(+)F4/80(+)) but not DCs (defined as CD11c(+)CD103(+)). LP macrophage frequency and number were decreased in two strains of CX3CR1-knockout mice and in mice deficient in the CX3CR1 ligand CX3CL1. All these knockout strains displayed markedly increased translocation of commensal bacteria to mLNs. Additionally, the severity of DSS-induced colitis was dramatically enhanced in the knockout mice as compared with controls. Disease severity could be limited by either administration of neutralizing IL-17A antibodies or transfer of CX3CR1-sufficient macrophages. Our data thus suggest key roles for the CX3CR1/CX3CL1 axis in the intestinal mucosa; further clarification of CX3CR1 function will likely direct efforts toward therapeutic intervention for mucosal inflammatory disorders such as IBD.

              </p>
            </td>
          </tr>
             
        </tbody></table>
            <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                <tr>
</tbody></table>
<table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
  <tr>
      
     
     
     <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
 <td style="padding:20px;width:25%;vertical-align:top">
   <div class="one">
    
     <img src='images/logo.jpg' width="160">
   </div>
   
	 
   </script>
 </td>
 <td style="padding:20px;width:75%;vertical-align:middle">
   <a style="text-decoration: none"  href="https://www.nature.com/articles/s41371-021-00555-y">
     <papertitle>Differentially expressed proteins in platelets derived from patients with hypertension</papertitle>
   </a>
   <br>
   <p>Armenta-Medina Y, Martínez-Vieyra I, Medina-Contreras O, Benitez-Cardoza CG, Jiménez-Pineda A, Reyes-López CA, Cerecedo D.<p>
   
   <br>
   <em>J Hum Hypertens </em>36(7), 640-50 &nbsp <font color="red"><strong>(10.1038/s41371-021-00555-y)</strong></font>
   <br>
 
   <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/34218268/">PubMed</a>
   
   <p></p>
   <p>
       <p style="text-align:justify">
       Hypertension (HTN) causes end-organ damage and is a major cause of morbidity and mortality globally. Recent studies suggested blood cells participate in the maintenance of HTN. Platelets-anucleated cell fragments derived from megakaryocytes-exert diverse functions, including their well-characterized role in the formation of hemostatic clots. However, platelets from patients with HTN exhibit altered membrane lipid and protein compositions that impact platelet function and lead to formation of aggregates and vascular obstructions. Here, for the first time, we have identified, by proteomic analyses, the most relevant 11 proteins that show the greatest difference in their expression in platelets derived from patients with HTN, in comparison with those from normotensive individuals. These proteins are involved in cytoskeletal organization and the coagulation cascade that contributes to platelet activation, release of granule contents, and aggregation, which culminate in thrombus formation. These results have important implications in our understanding of the molecular mechanisms associated with the development of HTN, and in consequence, the development of new strategies to counteract the cardiovascular disorders associated with constitutive activation of platelets in HTN.
             </p>
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
     
     <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#ffffff">
 <td style="padding:20px;width:25%;vertical-align:top">
   <div class="one">
    
     <img src='images/logo.jpg' width="160">
   </div>
   
	 
   </script>
 </td>
 <td style="padding:20px;width:75%;vertical-align:middle">
   <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1002/bem.22400">
     <papertitle>An Extremely Low-Frequency Vortex Magnetic Field Modifies Protein Expression, Rearranges the Cytoskeleton, and Induces Apoptosis of a Human Neuroblastoma Cell Line</papertitle>
   </a>
   <br>
   <p>Aparicio-Bautista DI, Chávez-Valenzuela D, Ambriz-Álvarez G, Córdova-Fraga T, Reyes-Grajeda JP, Medina-Contreras Ó, Rodríguez-Cruz F, García-Sierra F, Zúñiga-Sánchez P, Gutiérrez-Gutiérrez AM, Arellanes-Robledo J, Basurto-Islas G<p>
   
   <br>
   <em>Bioelectromagnetics </em>43(4), 225-44 &nbsp <font color="red"><strong>(10.1038/s41371-021-00555-y)</strong></font>
   <br>
 
   <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/35437793/">PubMed</a>
   
   <p></p>
   <p>
       <p style="text-align:justify">
       Homogeneous extremely low-frequency electromagnetic fields (ELF-EMFs) alter biological phenomena, including the cell phenotype and proliferation rate. Heterogenous vortex magnetic fields (VMFs), a new approach of exposure to magnetic fields, induce systematic movements on charged biomolecules from target cells; however, the effect of VMFs on living systems remains uncertain. Here, we designed, constructed, and characterized an ELF-VMF-modified Rodin's coil to expose SH-SY5Y cells. Samples were analyzed by performing 2D-differential-gel electrophoresis, identified by MALDI-TOF/TOF, validated by western blotting, and characterized by confocal microscopy. A total of 106 protein spots were differentially expressed; 40 spots were downregulated and 66 were upregulated in the exposed cell proteome, compared to the control cell proteome. The identified spots are associated with cytoskeleton and cell viability proteins, and according to the protein-protein interaction network, a significant interaction among them was found. Our data revealed a decrease in cell survival associated with apoptotic cells without effects on the cell cycle, as well as evident changes in the cytoskeleton. We demonstrated that ELF-VMFs, at a specific frequency and exposure time, alter the cell proteome and structurally affect the target cells. This is the first report showing that VMF application might be a versatile system for testing different hypotheses in living systems, using appropriate exposure parameters.
             </p>
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
			
                                    <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
 <td style="padding:20px;width:25%;vertical-align:top">
   <div class="one">
    
     <img src='images/frontiers.png' width="160">
   </div>
   </script>
 </td>
 <td style="padding:20px;width:75%;vertical-align:middle">
   <a style="text-decoration: none"  href="https://www.frontiersin.org/articles/10.3389/fcimb.2022.887647/full">
     <papertitle>Proteomic and Functional Analysis of the Effects of Quinoxaline Derivatives on <i>Entamoeba histolytica</i>.</papertitle>
   </a>
   <br>
   <p>Avila-Bonilla RG, López-Sandoval Á, Soto-Sánchez J, Marchat LA, Rivera G, Medina-Contreras O, Ramírez-Moreno E<p>
   
   <br>
   <em>Front Cell Infect Microbiol </em>12, 887647 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fcimb.2022.887647)</strong></font></span>
   <br>
 
   <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/35832378/">PubMed</a>
   
   <p></p>
   <p>
       <p style="text-align:justify">
       Quinoxalines are heterocyclic compounds that contain a benzene ring and a pyrazine ring. The oxidation of both nitrogen of the pyrazine ring results in quinoxaline derivatives (QdNO), which exhibit a variety of biological properties, including antiparasitic activity. However, its activity against Entamoeba histolytica, the protozoan that causes human amebiasis, is poorly understood. Recently, our group reported that various QdNOs produce morphological changes in E. histolytica trophozoites, increase reactive oxygen species, and inhibit thioredoxin reductase activity. Notably, T-001 and T-017 derivatives were among the QdNOs with the best activity. In order to contribute to the characterization of the antiamebic effect of QdNOs, in this work we analyzed the proteomic profile of E. histolytica trophozoites treated with the QdNOs T-001 and T-017, and the results were correlated with functional assays. A total number of 163 deregulated proteins were found in trophozoites treated with T-001, and 131 in those treated with T-017. A set of 21 overexpressed and 24 under-expressed proteins was identified, which were mainly related to cytoskeleton and intracellular traffic, nucleic acid transcription, translation and binding, and redox homeostasis. Furthermore, T-001 and T-017 modified the virulence of trophozoites, since they altered their erythrophagocytosis, migration, adhesion and cytolytic capacity. Our results show that in addition to alter reactive oxygen species, and thioredoxin reductase activity, T-001 and T-017 affect essential functions related to the actin cytoskeleton, which eventually affects E. histolytica virulence and survival.
       </p>
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
               
                        
                            <div class="citation">
                                <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#ffffff">
<td style="padding:20px;width:25%;vertical-align:top">
<div class="one">

 <img src='images/logo.jpg' width="160">
</div>
</script>
</td>
<td style="padding:20px;width:75%;vertical-align:middle">
<a style="text-decoration: none"  href="https://www.hindawi.com/journals/jir/2022/2909487/">
 <papertitle>Isthmin 1 is Expressed by Progenitor-Like Cells in the Lung: Phenotypical Analysis of Isthmin 1<sup>+</sup> Hematopoietic Stem-Like Cells in Homeostasis and during Infection</papertitle>
</a>
<br>
<p>Rivera-Torruco G, Martínez-Mendiola CA, Angeles-Floriano T, Jaimes-Ortega GA, Maravillas-Montero JL, García-Contreras R, González Y, Juárez E, Nava P, Ortiz-Navarrete V, Medina-Contreras O, Licona-Limón P, Valle-Rios R<p>

<br>
<em>J Immunol Res </em>2022, 2909487 &nbsp <span class="doi"><font color="red"><strong>(10.1155/2022/2909487)</strong></font></span>
<br>

<a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/35402623/">PubMed</a>

<p></p>
<p>
   <p style="text-align:justify">
   The process by which blood cells are generated has been widely studied in homeostasis and during pathogen-triggered inflammatory response. Recently, murine lungs have been shown to be a significant source of hematopoietic progenitors in a process known as extramedullary hematopoiesis. Using multiparametric flow cytometry, we have identified mesenchymal, endothelial, and hematopoietic progenitor cells that express the secreted small protein Isthmin 1 (ISM1). Further characterization of hematopoietic progenitor cells indicated that ISM1+ Lineage- Sca-1+ c-kit+ (ISM1+ LSK) cells are enriched in short-term hematopoietic stem cells (ST-HSCs). Moreover, most Sca-1+ ISM1+ cells express the residence marker CD49a, and this correlated with their localization in the extravascular region of the lung, indicating that ISM1+ cells are lung-resident cells. We also observed that ISM1+ cells express TLR4, TLR5, and TLR9, and, in a mouse model of sepsis induced by P. aeruginosa, we observed that all the LSK and ISM1+LSK cells were affected. We conclude that ISM1 is a novel biomarker associated with progenitor-like cells. ISM1+ cells are involved in the response to a bacterial challenge, suggesting an association between ISM1-producing cells and dangerous inflammatory responses like sepsis.
   </p>
        </td>
      </tr>
    </tbody></table>
    <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
        
        
        
                                          .
                                          <div class="citation">
                                              <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
              <td style="padding:20px;width:25%;vertical-align:top">
              <div class="one">

               <img src='images/logo.jpg' width="160">
              </div>
              </script>
              </td>
              <td style="padding:20px;width:75%;vertical-align:middle">
              <a style="text-decoration: none"  href="https://www.sciencedirect.com/science/article/abs/pii/S0882401022001073?via%3Dihub">
                  <p style="text-align:justify">
               <papertitle>Genomic characterization of two bacteriophages (vB_EcoS-phiEc3 and vB_EcoS-phiEc4) belonging to the genus Kagunavirus with lytic activity against uropathogenic <i>Escherichia coli</i></papertitle>
              </a>
              <br>
              <span class="authors">
                  <p>Rodea GE, González-Villalobos E, Medina-Contreras O, Castelán-Sánchez HG, Aguilar-Rodea P, Velázquez-Guadarrama N, Hernández-Chiñas U, Eslava-Campos CA, Balcázar JL, Molina-López J<p>

              <br>
              </a><span class="source"></span>
              <em>Microb Pathog </em>165, 105494 &nbsp <span class="doi"><font color="red"><strong>(10.1016/j.micpath.2022.105494)</strong></font></span>
              <br>

              <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/35314281/">PubMed</a>

              <p></p>
              <p>
                 <p style="text-align:justify">
                 In this study, the genomes of two lytic bacteriophages, vB_EcoS-phiEc3 and vB_EcoS-phiEc4, were sequenced and characterized using bioinformatics approaches. Whole-genome analysis showed that both phages belonged to the Kagunavirus genus, Guernseyvirinae subfamily and Siphoviridae family. Moreover, their genomes had 45, 288 bp and 44,540 bp, and G + C content of 48.42% and 50.04%, respectively. The genome of vB_EcoS-phiEc3 harbored 80 protein coding sequences (CDSs), whereas vB_EcoS-phiEc4 harbored 75 CDSs. Among them, 50 CDSs in vB_EcoS-phiEc3 and 44 CDSs in vB_EcoS-phiEc4 were considered as functional genes. Their lytic activity against multidrug-resistant uropathogenic Escherichia coli (UPEC) strains, as well as the absence of antibiotic resistance genes, lysogenic and virulence genes, enable vB_EcoS-phiEc3 and vB_EcoS-phiEc4 as a safe therapy option against UPEC infections.
                 </p>
                      </td>
                    </tr>
                  </tbody></table>
                  <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                    
                                    
                                    
                                    


                                
                                    



                                
                                    


                                    <div class="citation">
                                        <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#ffffff">
        <td style="padding:20px;width:25%;vertical-align:top">
        <div class="one">

         <img src='images/logo.jpg' width="160">
        </div>
        </script>
        </td>
        <td style="padding:20px;width:75%;vertical-align:middle">
        <a style="text-decoration: none"  href="https://www.liebertpub.com/doi/10.1089/jir.2021.0173">
         <papertitle>Interleukin-1 Receptor-Like 2: One Receptor, Three Agonists, and Many Implications</papertitle>
        </a>
        <br>
        <span class="authors">
            <p>Manzanares-Meza LD, Valle-Rios R, Medina-Contreras O<p>

        <br>
        </a><span class="source"></span>
        <em>J Interferon Cytokine Res </em>42(2), 49-61 &nbsp <span class="doi"><font color="red"><strong>(10.1089/jir.2021.0173)</strong></font></span>
        <br>

        <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/35171706/">PubMed</a>

        <p></p>
        <p>
           <p style="text-align:justify">
           The interleukin (IL)-1 superfamily of cytokines comprises 11 pro- and anti-inflammatory cytokines, which play essential roles during the immune response. Several pathogenic pathways are initiated by IL-1RL2 (interleukin 1 receptor-like 2) signaling, also known as IL-36R, in the skin, lungs, and gut. IL-36 cytokines promote the secretion of proinflammatory cytokines and chemokines, upregulation of antimicrobial peptides, proliferation mediators, and adhesion molecules on endothelial cells. In addition, the IL-36-IL-1RL2 axis has an essential role against viral infections, including a potential role in COVID-19 pathology. The evidence presented in this review highlights the importance of the axis IL-36-IL-1RL2 in the development of several inflammation-related diseases and the healing process. It suggests that IL-1RL2 ligands have specific roles depending on the tissue or cell source. However, there is still much to discover about this cytokine family, their functions in other organs, and how they accomplish a dual effect in inflammation and healing
           </p>
                </td>
              </tr>
            </tbody></table>
            <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                       


                                
                                    


                                    <div class="citation">
                                        <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
        <td style="padding:20px;width:25%;vertical-align:top">
        <div class="one">

         <img src='images/logo.jpg' width="160">
        </div>
        </script>
        </td>
        <td style="padding:20px;width:75%;vertical-align:middle">
        <a style="text-decoration: none"  href="https://www.jove.com/es/t/62204/evaluation-quantification-micro-epithelial-gaps-colonic-mucosa-using">
         <papertitle>Evaluation and Quantification of Micro Epithelial Gaps in the Colonic Mucosa using Immunofluorescence Staining</papertitle>
        </a>
        <br>
        <span class="authors">
            <p>Castro-Martínez F, Encarnación-García MDR, Candelario-Martinez A, Medina-Contreras O, Patiño-Lopez G, Schnoor M, Nava P<p>

        <br>
        </a><span class="source"></span>
        <em>J Vis Exp </em>2021(172),  &nbsp <span class="doi"><font color="red"><strong>(10.3791/62204)</strong></font></span>
        <br>

        <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/34180884/">PubMed</a>

        <p></p>
        <p>
           <p style="text-align:justify">
           Epithelial cells lining the intestinal mucosa create a physical barrier that separates the luminal content from the interstitium. Epithelial barrier impairment has been associated with the development of various pathologies such as inflammatory bowel diseases (IBD). In the inflamed mucosa, superficial erosions or micro-erosions that corrupt epithelial monolayers correspond to sites of high permeability. Several mechanisms have been implicated in the formation of micro-erosions including cell shedding and apoptosis. These micro-erosions often represent microscopic epithelial gaps randomly distributed in the colon. Visualization and quantification of those epithelial gaps has emerged as an important tool to investigate intestinal epithelial barrier function. Here, we describe a new method to visualize the specific location of where transcellular and paracellular permeability is enhanced in the inflamed colonic mucosa. In this assay, we apply a 10 kDa fluorescent dye conjugated to a lysine fixable dextran to visualize high permeability regions (HPR) in the colonic mucosa. Additional use of cell death markers revealed that HPR encompass apoptotic foci where epithelial extrusion/shedding occurs. The protocol described here provides a simple but effective approach to visualize and quantify micro-erosions in the intestine, which is a very useful tool in disease models, in which the intestinal epithelial barrier is compromised.
           </p>
                </td>
              </tr>
            </tbody></table>
            <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                       
                                       <div class="citation">
                                           <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#ffffff">
           <td style="padding:20px;width:25%;vertical-align:top">
           <div class="one">

            <img src='images/logo.jpg' width="160">
           </div>
           </script>
           </td>
           <td style="padding:20px;width:75%;vertical-align:middle">
           <a style="text-decoration: none"  href="https://www.mdpi.com/1422-0067/22/8/3882">
            <papertitle>Comparative Transcriptomes of the Body Wall of Wild and Farmed Sea Cucumber <i>Isostichopus badionotus</i></papertitle>
           </a>
           <br>
           <span class="authors">
               <p>Martín-Hernández R, Rodríguez-Canul R, Kantún-Moreno N, Olvera-Novoa MA, Medina-Contreras O, Garikoitz-Legarda C, Triviño JC, Zamora-Briseño JA, May-Solis V, Poot-Salazar A, Pérez-Vega JA, Gil-Zamorano J, Grant G, Dávalos A, Olivera-Castillo L<p>

           <br>
           </a><span class="source"></span>
           <em>Int J Mol Sci </em>22(8), 3882 &nbsp <span class="doi"><font color="red"><strong>(10.3390/ijms22083882)</strong></font></span>
           <br>

           <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/33918680/">PubMed</a>

           <p></p>
           <p>
              <p style="text-align:justify">
              Overfishing of sea cucumber Isostichopus badionotus from Yucatan has led to a major population decline. They are being captured as an alternative to traditional species despite a paucity of information about their health-promoting properties. The transcriptome of the body wall of wild and farmed I. badionotus has now been studied for the first time by an RNA-Seq approach. The functional profile of wild I. badionotus was comparable with data in the literature for other regularly captured species. In contrast, the metabolism of first generation farmed I. badionotus was impaired. This had multiple possible causes including a sub-optimal growth environment and impaired nutrient utilization. Several key metabolic pathways that are important in effective handling and accretion of nutrients and energy, or clearance of harmful cellular metabolites, were disrupted or dysregulated. For instance, collagen mRNAs were greatly reduced and deposition of collagen proteins impaired. Wild I. badionotus is, therefore, a suitable alternative to other widely used species but, at present, the potential of farmed I. badionotus is unclear. The environmental or nutritional factors responsible for their impaired function in culture remain unknown, but the present data gives useful pointers to the underlying problems associated with their aquaculture.
              </p>
                   </td>
                 </tr>
               </tbody></table>
               <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>


                                
                                <div class="citation">
                                    <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#ffffff">
    <td style="padding:20px;width:25%;vertical-align:top">
    <div class="one">

     <img src='images/logo.jpg' width="160">
    </div>
    </script>
    </td>
    <td style="padding:20px;width:75%;vertical-align:middle">
    <a style="text-decoration: none"  href="https://www.pnas.org/doi/full/10.1073/pnas.2004484117">
     <papertitle>IL-36R signaling integrates innate and adaptive immune-mediated protection against enteropathogenic bacteria</papertitle>
    </a>
    <br>
    <span class="authors">
        <p>Ngo VL, Abo H, Kuczma M, Szurek E, Moore N, Medina-Contreras O, Nusrat A, Merlin D, Gewirtz AT, Ignatowicz L, Denning TL<p>

    <br>
    </a><span class="source"></span>
    <em>Proc Natl Acad Sci USA </em>117(44), 27540-8 &nbsp <span class="doi"><font color="red"><strong>(10.1073/pnas.2004484117)</strong></font></span>
    <br>

    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/33918680/">PubMed</a>

    <p></p>
    <p>
       <p style="text-align:justify">
       Enteropathogenic bacterial infections are a global health issue associated with high mortality, particularly in developing countries. Efficient host protection against enteropathogenic bacterial infection is characterized by coordinated responses between immune and nonimmune cells. In response to infection in mice, innate immune cells are activated to produce interleukin (IL)-23 and IL-22, which promote antimicrobial peptide (AMP) production and bacterial clearance. IL-36 cytokines are proinflammatory IL-1 superfamily members, yet their role in enteropathogenic bacterial infection remains poorly defined. Using the enteric mouse pathogen, C.rodentium, we demonstrate that signaling via IL-36 receptor (IL-36R) orchestrates a crucial innate-adaptive immune link to control bacterial infection. IL-36R-deficient mice (Il1rl2-/- ) exhibited significant impairment in expression of IL-22 and AMPs, increased intestinal damage, and failed to contain C. rodentium compared to controls. These defects were associated with failure to induce IL-23 and IL-6, two key IL-22 inducers in the early and late phases of infection, respectively. Treatment of Il1rl2-/- mice with IL-23 during the early phase of C. rodentium infection rescued IL-22 production from group 3 innate lymphoid cells (ILCs), whereas IL-6 administration during the late phase rescued IL-22-mediated production from CD4+ T cell, and both treatments protected Il1rl2-/- mice from uncontained infection. Furthermore, IL-36R-mediated IL-22 production by CD4+ T cells was dependent upon NFκB-p65 and IL-6 expression in dendritic cells (DCs), as well as aryl hydrocarbon receptor (AhR) expression by CD4+ T cells. Collectively, these data demonstrate that the IL-36 signaling pathway integrates innate and adaptive immunity leading to host defense against enteropathogenic bacterial infection.
       </p>
            </td>
          </tr>
        </tbody></table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>


            <div class="citation">
                <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
<td style="padding:20px;width:25%;vertical-align:top">
<div class="one">

<img src='images/logo.jpg' width="160">
</div>
</script>
</td>
<td style="padding:20px;width:75%;vertical-align:middle">
<a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
<papertitle>Nucleoredoxin interaction with flightless-I/actin complex is differentially altered in alcoholic liver disease</papertitle>
</a>
<br>
<span class="authors">
<p>Alarcón-Sánchez BR, Guerrero-Escalera D, Rosas-Madrigal S, Ivette Aparicio-Bautista D, Reyes-Gordillo K, Lakshman MR, Ortiz-Fernández A, Quezada H, Medina-Contreras O, Villa-Treviño S, Isael Pérez-Carreón J, Arellanes-Robledo J<p>

<br>
</a><span class="source"></span>
<em>Basic Clin Pharmacol Toxicol </em>127(5), 389-404 &nbsp <span class="doi"><font color="red"><strong>(10.1111/bcpt.13451)</strong></font></span>
<br>

<a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a>

<p></p>
<p>
<p style="text-align:justify">
Alcoholic liver disease (ALD) may be attributed to multiple hits driving several alterations. The aim of this work was to determine whether nucleoredoxin (NXN) interacts with flightless-I (FLII)/actin complex and how this ternary complex is altered during ALD progression induced by different ALD models. ALD was recapitulated in C57BL/6J female mice by the well-known ALD Lieber-DeCarli model, and by an in vitro human co-culture system overexpressing NXN. The effects of ethanol and low doses of lipopolysaccharides (LPS) and diethylnitrosamine (DEN) were also evaluated in vivo as a first approach of an ALD multi-hit protocol. We demonstrated that NXN interacts with FLII/actin complex. This complex was differentially altered in ALD in vivo and in vitro, and NXN overexpression partially reverted this alteration. We also showed that ethanol, LPS and DEN synergistically induced liver structural disarrangement, steatosis and inflammatory infiltration accompanied by increased levels of proliferation (Ki67), ethanol metabolism (CYP2E1), hepatocarcinogenesis (GSTP1) and LPS-inducible (MYD88 and TLR4) markers. In summary, we provide evidence showing that NXN/FLII/actin complex is involved in ALD progression and that NXN might be involved in the regulation of FLII/actin-dependent cellular functions. Moreover, we present a promising first approach of a multi-hit protocol to better recapitulate ALD pathogenesis.
</p>
</td>
</tr>
</tbody></table>
<table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        


                                
                                    

                                    
                      
                        <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                        <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                        </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                        <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                        <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                        <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                        </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                        <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                        The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                        </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                              
                                    


                                
                                    
                                    <!---

                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="29781368"
                    
                                      Villagomez FR, Medina-Contreras O, Cerna-Cortes JF, Patino-Lopez G.
                                 
                                          The role of the oncogenic Rab35 in cancer invasion, metastasis, and immune evasion, especially in leukemia.
                                      </a><span class="source">Small GTPases</span>.
                                  
                                  
                                     <span class="volume">11</span><span class="issue">(5)</span><span class="pages">:334-345</span><span>. </span><span class="doi">doi: 10.1080/21541248.2018.1463895. </span><span 


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="32517205"
                                    
                                  >
                                  <label for="cit64022341">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Olivera-Castillo L, Grant G, Kantún-Moreno N, Barrera-Pérez HA, Montero J, Olvera-Novoa MA, Carrillo-Cocom LM, Acevedo JJ, Puerto-Castillo C, May Solís V, Pérez-Vega JA, Gil-Zamorano J, Hernández-Garibay E, Fernández-Herrera MA, Pérez-Tapia M, Medina-Contreras O, Villanueva-Toledo JR, Rodriguez-Canul R, Dávalos A.
                                  </span><a style="text-decoration: none"  href="/pubmed/32517205/">
                                          A Glycosaminoglycan-Rich Fraction from Sea Cucumber <i>Isostichopus badionotus</i> Has Potent Anti-Inflammatory Properties In Vitro and In Vivo.
                                      </a><span class="source">Nutrients</span>.
                                  
                                  
                                      <span class="pubdate">2020 Jun 6;</span><span class="volume">12</span><span class="issue">(6)</span><span>. </span><span class="doi">doi: 10.3390/nu12061698. </span><span class="pmid">PubMed PMID:
                                      32517205</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC7352476</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="32373066"
                                    
                                  >
                                  <label for="cit64022342">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Alvarez-Herrera S, Escamilla R, Medina-Contreras O, Saracco R, Flores Y, Hurtado-Alvarado G, Maldonado-García JL, Becerril-Villanueva E, Pérez-Sánchez G, Pavón L.
                                  </span><a style="text-decoration: none"  href="/pubmed/32373066/">
                                          Immunoendocrine Peripheral Effects Induced by Atypical Antipsychotics.
                                      </a><span class="source">Front Endocrinol (Lausanne)</span>.
                                  
                                  
                                      <span class="pubdate">2020;</span><span class="volume">11</span><span class="pages">:195</span><span>. </span><span class="doi">doi: 10.3389/fendo.2020.00195. </span><span class="pubstatus">eCollection 2020. </span><span class="pubtype">Review. </span><span class="pmid">PubMed PMID:
                                      32373066</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC7186385</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="31799578"
                                    
                                  >
                                  <label for="cit64022343">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Vásquez-Procopio J, Osorio B, Cortés-Martínez L, Hernández-Hernández F, Medina-Contreras O, Ríos-Castro E, Comjean A, Li F, Hu Y, Mohr S, Perrimon N, Missirlis F.
                                  </span><a style="text-decoration: none"  href="/pubmed/31799578/">
                                          Intestinal response to dietary manganese depletion in Drosophila.
                                      </a><span class="source">Metallomics</span>.
                                  
                                  
                                      <span class="pubdate">2020 Feb 1;</span><span class="volume">12</span><span class="issue">(2)</span><span class="pages">:218-240</span><span>. </span><span class="doi">doi: 10.1039/c9mt00218a. </span><span class="pubstatus">Epub 2019 Dec 4. </span><span class="pmid">PubMed PMID:
                                      31799578</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="33064687"
                                    
                                  >
                                  <label for="cit64022338">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Medina-Contreras O, Luvián-Morales J, Valdez-Palomares F, Flores-Cisneros L, Sánchez-López MS, Soto-Lugo JH, Castro-Eguiluz D.
                                  </span><a style="text-decoration: none"  href="/pubmed/33064687/">
                                          IMMUNONUTRITION IN CERVICAL CANCER: IMMUNE RESPONSE MODULATION BY DIET.
                                      </a><span class="source">Rev Invest Clin</span>.
                                  
                                  
                                      <span class="pubdate">2020;</span><span class="volume">72</span><span class="issue">(4)</span><span class="pages">:219-230</span><span>. </span><span class="doi">doi: 10.24875/RIC.20000062. </span><span class="pubtype">Review. </span><span class="pmid">PubMed PMID:
                                      33064687</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="33064680"
                                    
                                  >
                                  <label for="cit64022339">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Manzanares-Meza LD, Medina-Contreras O.
                                  </span><a style="text-decoration: none"  href="/pubmed/33064680/">
                                          SARS-CoV-2 and influenza: a comparative overview and treatment implications.
                                      </a><span class="source">Bol Med Hosp Infant Mex</span>.
                                  
                                  
                                      <span class="pubdate">2020;</span><span class="volume">77</span><span class="issue">(5)</span><span class="pages">:262-273</span><span>. </span><span class="doi">doi: 10.24875/BMHIM.20000183. </span><span class="pubtype">Review. </span><span class="pmid">PubMed PMID:
                                      33064680</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="31678145"
                                    
                                  >
                                  <label for="cit64022344">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Calleja LF, Belmont-Díaz JA, Medina-Contreras O, Quezada H, Yoval-Sánchez B, Campos-García J, Rodríguez-Zavala JS.
                                  </span><a style="text-decoration: none"  href="/pubmed/31678145/">
                                          Omeprazole as a potent activator of human cytosolic aldehyde dehydrogenase ALDH1A1.
                                      </a><span class="source">Biochim Biophys Acta Gen Subj</span>.
                                  
                                  
                                      <span class="pubdate">2020 Jan;</span><span class="volume">1864</span><span class="issue">(1)</span><span class="pages">:129451</span><span>. </span><span class="doi">doi: 10.1016/j.bbagen.2019.129451. </span><span class="pubstatus">Epub 2019 Oct 31. </span><span class="pmid">PubMed PMID:
                                      31678145</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="31354816"
                                    
                                  >
                                  <label for="cit64022345">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Jiménez-Osorio AS, Aguilar-Lucio AO, Cárdenas-Hernández H, Musalem-Younes C, Solares-Tlapechco J, Costa-Urrutia P, Medina-Contreras O, Granados J, Rodríguez-Arellano ME.
                                  </span><a style="text-decoration: none"  href="/pubmed/31354816/">
                                          Polymorphisms in Adipokines in Mexican Children with Obesity.
                                      </a><span class="source">Int J Endocrinol</span>.
                                  
                                  
                                      <span class="pubdate">2019;</span><span class="volume">2019</span><span class="pages">:4764751</span><span>. </span><span class="doi">doi: 10.1155/2019/4764751. </span><span class="pubstatus">eCollection 2019. </span><span class="pmid">PubMed PMID:
                                      31354816</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC6634012</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="30802270"
                                    
                                  >
                                  <label for="cit64022346">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Costa-Urrutia P, Vizuet-Gámez A, Ramirez-Alcántara M, Guillen-González MÁ, Medina-Contreras O, Valdes-Moreno M, Musalem-Younes C, Solares-Tlapechco J, Granados J, Franco-Trecu V, Rodriguez-Arellano ME.
                                  </span><a style="text-decoration: none"  href="/pubmed/30802270/">
                                          Obesity measured as percent body fat, relationship with body mass index, and percentile curves for Mexican pediatric population.
                                      </a><span class="source">PLoS One</span>.
                                  
                                  
                                      <span class="pubdate">2019;</span><span class="volume">14</span><span class="issue">(2)</span><span class="pages">:e0212792</span><span>. </span><span class="doi">doi: 10.1371/journal.pone.0212792. </span><span class="pubstatus">eCollection 2019. </span><span class="pmid">PubMed PMID:
                                      30802270</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC6388924</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="30764545"
                                    
                                  >
                                  <label for="cit64022347">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Jiménez-Osorio AS, Musalem-Younes C, Cárdenas-Hernández H, Solares-Tlapechco J, Costa-Urrutia P, Medina-Contreras O, Granados J, López-Saucedo C, Estrada-Garcia T, Rodríguez-Arellano ME.
                                  </span><a style="text-decoration: none"  href="/pubmed/30764545/">
                                          Common Polymorphisms Linked to Obesity and Cardiovascular Disease in Europeans and Asians are Associated with Type 2 Diabetes in Mexican Mestizos.
                                      </a><span class="source">Medicina (Kaunas)</span>.
                                  
                                  
                                      <span class="pubdate">2019 Feb 5;</span><span class="volume">55</span><span class="issue">(2)</span><span>. </span><span class="doi">doi: 10.3390/medicina55020040. </span><span class="pmid">PubMed PMID:
                                      30764545</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC6410269</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="30687322"
                                    
                                  >
                                  <label for="cit64022348">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Piedra-Quintero ZL, Serrano C, Villegas-Sepúlveda N, Maravillas-Montero JL, Romero-Ramírez S, Shibayama M, Medina-Contreras O, Nava P, Santos-Argumedo L.
                                  </span><a style="text-decoration: none"  href="/pubmed/30687322/">
                                          Myosin 1F Regulates M1-Polarization by Stimulating Intercellular Adhesion in Macrophages.
                                      </a><span class="source">Front Immunol</span>.
                                  
                                  
                                      <span class="pubdate">2018;</span><span class="volume">9</span><span class="pages">:3118</span><span>. </span><span class="doi">doi: 10.3389/fimmu.2018.03118. </span><span class="pubstatus">eCollection 2018. </span><span class="pmid">PubMed PMID:
                                      30687322</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC6335276</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="30319653"
                                    
                                  >
                                  <label for="cit64022349">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Antonio-Herrera L, Badillo-Godinez O, Medina-Contreras O, Tepale-Segura A, García-Lozano A, Gutierrez-Xicotencatl L, Soldevila G, Esquivel-Guadarrama FR, Idoyaga J, Bonifaz LC.
                                  </span><a style="text-decoration: none"  href="/pubmed/30319653/">
                                          The Nontoxic Cholera B Subunit Is a Potent Adjuvant for Intradermal DC-Targeted Vaccination.
                                      </a><span class="source">Front Immunol</span>.
                                  
                                  
                                      <span class="pubdate">2018;</span><span class="volume">9</span><span class="pages">:2212</span><span>. </span><span class="doi">doi: 10.3389/fimmu.2018.02212. </span><span class="pubstatus">eCollection 2018. </span><span class="pmid">PubMed PMID:
                                      30319653</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC6171476</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="29760082"
                                    
                                  >
                                  <label for="cit64022351">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Ngo VL, Abo H, Maxim E, Harusato A, Geem D, Medina-Contreras O, Merlin D, Gewirtz AT, Nusrat A, Denning TL.
                                  </span><a style="text-decoration: none"  href="/pubmed/29760082/">
                                          A cytokine network involving IL-36γ, IL-23, and IL-22 promotes antimicrobial defense and recovery from intestinal barrier damage.
                                      </a><span class="source">Proc Natl Acad Sci U S A</span>.
                                  
                                  
                                      <span class="pubdate">2018 May 29;</span><span class="volume">115</span><span class="issue">(22)</span><span class="pages">:E5076-E5085</span><span>. </span><span class="doi">doi: 10.1073/pnas.1718902115. </span><span class="pubstatus">Epub 2018 May 14. </span><span class="pmid">PubMed PMID:
                                      29760082</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC5984499</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="29382489"
                                    
                                  >
                                  <label for="cit64022352">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Manzanares-Meza LD, Gutiérrez-Román CI, Medina-Contreras O.
                                  </span><a style="text-decoration: none"  href="/pubmed/29382489/">
                                          MALDI imaging: beyond classic diagnosis.
                                      </a><span class="source">Bol Med Hosp Infant Mex</span>.
                                  
                                  
                                      <span class="pubdate">2017 May - Jun;</span><span class="volume">74</span><span class="issue">(3)</span><span class="pages">:212-218</span><span>. </span><span class="doi">doi: 10.1016/j.bmhimx.2017.03.004. </span><span class="pubstatus">Epub 2017 May 23. </span><span class="pubtype">Review. </span><span class="pmid">PubMed PMID:
                                      29382489</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="29382485"
                                    
                                  >
                                  <label for="cit64022354">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Guzmán-Ortiz AL, Aparicio-Ozores G, Valle-Rios R, Medina-Contreras O, Patiño-López G, Quezada H.
                                  </span><a style="text-decoration: none"  href="/pubmed/29382485/">
                                          Proteomic changes in a childhood acute lymphoblastic leukemia cell line during the adaptation to vincristine.
                                      </a><span class="source">Bol Med Hosp Infant Mex</span>.
                                  
                                  
                                      <span class="pubdate">2017 May - Jun;</span><span class="volume">74</span><span class="issue">(3)</span><span class="pages">:181-192</span><span>. </span><span class="doi">doi: 10.1016/j.bmhimx.2017.03.005. </span><span class="pubstatus">Epub 2017 May 23. </span><span class="pmid">PubMed PMID:
                                      29382485</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="29382486"
                                    
                                  >
                                  <label for="cit64022353">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Camacho-Millán R, Aguilar-Medina EM, Quezada H, Medina-Contreras Ó, Patiño-López G, Cárdenas-Cota HM, Ramos-Payán R.
                                  </span><a style="text-decoration: none"  href="/pubmed/29382486/">
                                          Characterization of Cry toxins from autochthonous Bacillus thuringiensis isolates from Mexico.
                                      </a><span class="source">Bol Med Hosp Infant Mex</span>.
                                  
                                  
                                      <span class="pubdate">2017 May - Jun;</span><span class="volume">74</span><span class="issue">(3)</span><span class="pages">:193-199</span><span>. </span><span class="doi">doi: 10.1016/j.bmhimx.2017.03.002. </span><span class="pubstatus">Epub 2017 May 10. </span><span class="pmid">PubMed PMID:
                                      29382486</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="27538402"
                                    
                                  >
                                  <label for="cit64022355">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Hernández-Trejo JA, Suárez-Pérez D, Gutiérrez-Martínez IZ, Fernandez-Vargas OE, Serrano C, Candelario-Martínez AA, Meraz-Ríos MA, Citalán-Madrid AF, Hernández-Ruíz M, Reyes-Maldonado E, Valle-Rios R, Feintuch-Unger JH, Schnoor M, Villegas-Sepúlveda N, Medina-Contreras O, Nava P.
                                  </span><a style="text-decoration: none"  href="/pubmed/27538402/">
                                          The pro-inflammatory cytokines IFNγ/TNFα increase chromogranin A-positive neuroendocrine cells in the colonic epithelium.
                                      </a><span class="source">Biochem J</span>.
                                  
                                  
                                      <span class="pubdate">2016 Nov 1;</span><span class="volume">473</span><span class="issue">(21)</span><span class="pages">:3805-3818</span><span>. </span><span class="doi">doi: 10.1042/BCJ20160390. </span><span class="pubstatus">Epub 2016 Aug 18. </span><span class="pmid">PubMed PMID:
                                      27538402</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="26846144"
                                    
                                  >
                                  <label for="cit64022356">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Gómez-Suárez M, Gutiérrez-Martínez IZ, Hernández-Trejo JA, Hernández-Ruiz M, Suárez-Pérez D, Candelario A, Kamekura R, Medina-Contreras O, Schnoor M, Ortiz-Navarrete V, Villegas-Sepúlveda N, Parkos C, Nusrat A, Nava P.
                                  </span><a style="text-decoration: none"  href="/pubmed/26846144/">
                                          14-3-3 Proteins regulate Akt Thr308 phosphorylation in intestinal epithelial cells.
                                      </a><span class="source">Cell Death Differ</span>.
                                  
                                  
                                      <span class="pubdate">2016 Jun;</span><span class="volume">23</span><span class="issue">(6)</span><span class="pages">:1060-72</span><span>. </span><span class="doi">doi: 10.1038/cdd.2015.163. </span><span class="pubstatus">Epub 2016 Feb 5. </span><span class="pmid">PubMed PMID:
                                      26846144</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC4987727</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="26590314"
                                    
                                  >
                                  <label for="cit64022357">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Medina-Contreras O, Harusato A, Nishio H, Flannigan KL, Ngo V, Leoni G, Neumann PA, Geem D, Lili LN, Ramadas RA, Chassaing B, Gewirtz AT, Kohlmeier JE, Parkos CA, Towne JE, Nusrat A, Denning TL.
                                  </span><a style="text-decoration: none"  href="/pubmed/26590314/">
                                          Cutting Edge: IL-36 Receptor Promotes Resolution of Intestinal Damage.
                                      </a><span class="source">J Immunol</span>.
                                  
                                  
                                      <span class="pubdate">2016 Jan 1;</span><span class="volume">196</span><span class="issue">(1)</span><span class="pages">:34-8</span><span>. </span><span class="doi">doi: 10.4049/jimmunol.1501312. </span><span class="pubstatus">Epub 2015 Nov 20. </span><span class="pmid">PubMed PMID:
                                      26590314</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC4684965</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="25079689"
                                    
                                  >
                                  <label for="cit64022358">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Nava P, Kamekura R, Quirós M, Medina-Contreras O, Hamilton RW, Kolegraff KN, Koch S, Candelario A, Romo-Parra H, Laur O, Hilgarth RS, Denning TL, Parkos CA, Nusrat A.
                                  </span><a style="text-decoration: none"  href="/pubmed/25079689/">
                                          IFNγ-induced suppression of β-catenin signaling: evidence for roles of Akt and 14.3.3ζ.
                                      </a><span class="source">Mol Biol Cell</span>.
                                  
                                  
                                      <span class="pubdate">2014 Oct 1;</span><span class="volume">25</span><span class="issue">(19)</span><span class="pages">:2894-904</span><span>. </span><span class="doi">doi: 10.1091/mbc.E13-09-0512. </span><span class="pubstatus">Epub 2014 Jul 30. </span><span class="pmid">PubMed PMID:
                                      25079689</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC4230580</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="24621992"
                                    
                                  >
                                  <label for="cit64022360">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Weber DA, Sumagin R, McCall IC, Leoni G, Neumann PA, Andargachew R, Brazil JC, Medina-Contreras O, Denning TL, Nusrat A, Parkos CA.
                                  </span><a style="text-decoration: none"  href="/pubmed/24621992/">
                                          Neutrophil-derived JAML inhibits repair of intestinal epithelial injury during acute inflammation.
                                      </a><span class="source">Mucosal Immunol</span>.
                                  
                                  
                                      <span class="pubdate">2014 Sep;</span><span class="volume">7</span><span class="issue">(5)</span><span class="pages">:1221-32</span><span>. </span><span class="doi">doi: 10.1038/mi.2014.12. </span><span class="pubstatus">Epub 2014 Mar 12. </span><span class="pmid">PubMed PMID:
                                      24621992</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC4340686</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="24899505"
                                    
                                  >
                                  <label for="cit64022359">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Geem D, Medina-Contreras O, McBride M, Newberry RD, Koni PA, Denning TL.
                                  </span><a style="text-decoration: none"  href="/pubmed/24899505/">
                                          Specific microbiota-induced intestinal Th17 differentiation requires MHC class II but not GALT and mesenteric lymph nodes.
                                      </a><span class="source">J Immunol</span>.
                                  
                                  
                                      <span class="pubdate">2014 Jul 1;</span><span class="volume">193</span><span class="issue">(1)</span><span class="pages">:431-8</span><span>. </span><span class="doi">doi: 10.4049/jimmunol.1303167. </span><span class="pubstatus">Epub 2014 Jun 4. </span><span class="pmid">PubMed PMID:
                                      24899505</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC4097179</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="22981539"
                                    
                                  >
                                  <label for="cit64022361">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Khounlotham M, Kim W, Peatman E, Nava P, Medina-Contreras O, Addis C, Koch S, Fournier B, Nusrat A, Denning TL, Parkos CA.
                                  </span><a style="text-decoration: none"  href="/pubmed/22981539/">
                                          Compromised intestinal epithelial barrier induces adaptive immune compensation that protects from colitis.
                                      </a><span class="source">Immunity</span>.
                                  
                                  
                                      <span class="pubdate">2012 Sep 21;</span><span class="volume">37</span><span class="issue">(3)</span><span class="pages">:563-73</span><span>. </span><span class="doi">doi: 10.1016/j.immuni.2012.06.017. </span><span class="pubstatus">Epub 2012 Sep 13. </span><span class="pmid">PubMed PMID:
                                      22981539</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC3564580</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="22644046"
                                    
                                  >
                                  <label for="cit64022362">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Geem D, Medina-Contreras O, Kim W, Huang CS, Denning TL.
                                  </span><a style="text-decoration: none"  href="/pubmed/22644046/">
                                          Isolation and characterization of dendritic cells and macrophages from the mouse intestine.
                                      </a><span class="source">J Vis Exp</span>.
                                  
                                  
                                      <span class="pubdate">2012 May 21;</span><span class="issue">(63)</span><span class="pages">:e4040</span><span>. </span><span class="doi">doi: 10.3791/4040. </span><span class="pmid">PubMed PMID:
                                      22644046</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC3466926</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="22045567"
                                    
                                  >
                                  <label for="cit64022363">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Medina-Contreras O, Geem D, Laur O, Williams IR, Lira SA, Nusrat A, Parkos CA, Denning TL.
                                  </span><a style="text-decoration: none"  href="/pubmed/22045567/">
                                          CX3CR1 regulates intestinal macrophage  homeostasis, bacterial translocation,  and colitogenic Th17 responses in mice.
                                      </a><span class="source">J Clin Invest</span>.
                                  
                                  
                                      <span class="pubdate">2011 Dec;</span><span class="volume">121</span><span class="issue">(12)</span><span class="pages">:4787-95</span><span>. </span><span class="doi">doi: 10.1172/JCI59150. </span><span class="pmid">PubMed PMID:
                                      22045567</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC3226003</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="21666057"
                                    
                                  >
                                  <label for="cit64022364">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Denning TL, Norris BA, Medina-Contreras O, Manicassamy S, Geem D, Madan R, Karp CL, Pulendran B.
                                  </span><a style="text-decoration: none"  href="/pubmed/21666057/">
                                          Functional specializations of intestinal dendritic cell and macrophage subsets that control Th17 and regulatory T cell responses are dependent on the T cell/APC ratio, source of mouse strain, and regional localization.
                                      </a><span class="source">J Immunol</span>.
                                  
                                  
                                      <span class="pubdate">2011 Jul 15;</span><span class="volume">187</span><span class="issue">(2)</span><span class="pages">:733-47</span><span>. </span><span class="doi">doi: 10.4049/jimmunol.1002701. </span><span class="pubstatus">Epub 2011 Jun 10. </span><span class="pmid">PubMed PMID:
                                      21666057</span><span class="pmcid">; PubMed Central PMCID:
                                          PMC3131424</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="20556794"
                                    
                                  >
                                  <label for="cit64022365">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Garay E, Patiño-López G, Islas S, Alarcón L, Canche-Pool E, Valle-Rios R, Medina-Contreras O, Granados G, Chávez-Munguía B, Juaristi E, Ortiz-Navarrete V, González-Mariscal L.
                                  </span><a style="text-decoration: none"  href="/pubmed/20556794/">
                                          CRTAM: A molecule involved in epithelial cell adhesion.
                                      </a><span class="source">J Cell Biochem</span>.
                                  
                                  
                                      <span class="pubdate">2010 Sep 1;</span><span class="volume">111</span><span class="issue">(1)</span><span class="pages">:111-22</span><span>. </span><span class="doi">doi: 10.1002/jcb.22673. </span><span class="pmid">PubMed PMID:
                                      20556794</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="19799932"
                                    
                                  >
                                  <label for="cit64022366">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Medina-Contreras O, Soldevila G, Patiño-Lopez G, Canche-Pool E, Valle-Rios R, Ortiz-Navarrete V.
                                  </span><a style="text-decoration: none"  href="/pubmed/19799932/">
                                          Role of CRTAM during mouse early T lymphocytes development.
                                      </a><span class="source">Dev Comp Immunol</span>.
                                  
                                  
                                      <span class="pubdate">2010 Feb;</span><span class="volume">34</span><span class="issue">(2)</span><span class="pages">:196-202</span><span>. </span><span class="doi">doi: 10.1016/j.dci.2009.09.009. </span><span class="pubstatus">Epub 2009 Oct 8. </span><span class="pmid">PubMed PMID:
                                      19799932</span><span>. </span>
                              </div>
                            </div>
                          </div>


                                
                                    


                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    
                                        pmid="19695707"
                                    
                                  >
                                  <label for="cit64022367">select</label>
                              </div>
                              <div class="ncbi-docsum">
                                <span class="authors">
                                      Valle-Rios R, Patiño-Lopez G, Medina-Contreras O, Canche-Pool E, Recillas-Targa F, Lopez-Bayghen E, Zlotnik A, Ortiz-Navarrete V.
                                  </span><a style="text-decoration: none"  href="/pubmed/19695707/">
                                          Characterization of CRTAM gene promoter: AP-1 transcription factor control its expression in human T CD8 lymphocytes.
                                      </a><span class="source">Mol Immunol</span>.
                                  
                                  
                                      <span class="pubdate">2009 Oct;</span><span class="volume">46</span><span class="issue">(16)</span><span class="pages">:3379-87</span><span>. </span><span class="doi">doi: 10.1016/j.molimm.2009.07.016. </span><span class="pubstatus">Epub 2009 Aug 19. </span><span class="pmid">PubMed PMID:
                                      19695707</span><span>. </span>
   
   
   
   
   
   
                                    <!---
                                    <div class="citation"> <tr onmouseout="dreamfusion_stop()" onmouseover="dreamfusion_start()"  bgcolor="#00FFFF">
                                    <td style="padding:20px;width:25%;vertical-align:top"> <div class="one"> <img src='images/frontiers.png' width="160">
                                    </div></script></td><td style="padding:20px;width:75%;vertical-align:middle">
                                    <a style="text-decoration: none"  href="https://onlinelibrary.wiley.com/doi/10.1111/bcpt.13451">
                                    <papertitle>Molecular Epidemiology of <i>Acinetobacter calcoaceticus</i>-<i>Acinetobacter baumannii</i> Complex Isolated From Children at the Hospital Infantil de México Federico Gómez</papertitle></a><br>
                                    <span class="authors"><p>Mancilla-Rojano J, Ochoa SA, Reyes-Grajeda JP, Flores V, Medina-Contreras O, Espinosa-Mazariego K, Parra-Ortega I, Rosa-Zamboni D, Castellanos-Cruz MDC, Arellano-Galindo J, Cevallos MA, Hernández-Castro R, Xicohtencatl-Cortes J, Cruz-Córdova A<p><br>
                                    </a><span class="source"></span><em>Front Microbiol </em>11, 576673 &nbsp <span class="doi"><font color="red"><strong>(10.3389/fmicb.2020.576673)</strong></font></span><br>
                                    <a style="text-decoration: none"  href="https://pubmed.ncbi.nlm.nih.gov/32524749/">PubMed</a><p></p><p><p style="text-align:justify">
                                    The Acinetobacter calcoaceticus-baumannii (Acb) complex is regarded as a group of phenotypically indistinguishable opportunistic pathogens responsible for mainly causing hospital-acquired pneumonia and bacteremia. The aim of this study was to determine the frequency of isolation of the species that constitute the Acb complex, as well as their susceptibility to antibiotics, and their distribution at the Hospital Infantil de Mexico Federico Gomez (HIMFG). A total of 88 strains previously identified by Vitek 2®, 40 as Acinetobacter baumannii and 48 as Acb complex were isolated from 52 children from 07, January 2015 to 28, September 2017. A. baumannii accounted for 89.77% (79/88) of the strains; Acinetobacter pittii, 6.82% (6/88); and Acinetobacter nosocomialis, 3.40% (3/88). Most strains were recovered mainly from patients in the intensive care unit (ICU) and emergency wards. Blood cultures (BC) provided 44.32% (39/88) of strains. The 13.63% (12/88) of strains were associated with primary bacteremia, 3.4% (3/88) with secondary bacteremia, and 2.3% (2/88) with pneumonia. In addition, 44.32% (39/88) were multidrug-resistant (MDR) strains and, 11.36% (10/88) were extensively drug-resistant (XDR). All strains amplified the bla OXA-51 gene; 51.13% (45/88), the bla OXA-23 gene; 4.54% (4/88), the bla OXA-24 gene; and 2.27% (2/88), the bla OXA-58 gene. Plasmid profiles showed that the strains had 1-6 plasmids. The strains were distributed in 52 pulsotypes, and 24 showed identical restriction patterns, with a correlation coefficient of 1.0. Notably, some strains with the same pulsotype were isolated from different patients, wards, or years, suggesting the persistence of more than one clone. Twenty-seven sequence types (STs) were determined for the strains based on a Pasteur multilocus sequence typing (MLST) scheme using massive sequencing; the most prevalent was ST 156 (27.27%, 24/88). The Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-Cas I-Fb system provided amplification in A. baumannii and A. pittii strains (22.73%, 20/88). This study identified an increased number of MDR strains and the relationship among strains through molecular typing. The data suggest that more than one strain could be causing an infection in some patient. The implementation of molecular epidemiology allowed the characterization of a set of strains and identification of different attributes associated with its distribution in a specific environment.
                                    </p></td></tr></tbody></table><table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
                                    --->
 
</html>
